addons/binding/org.openhab.binding.airquality/README.md

```diff
@@ -0,0 +1,118 @@
+
+The World Air Quality Index project is a social enterprise project started in 2007. Its mission is to promote Air Pollution awareness and provide a unified Air Quality information for the whole world. 
+
+The project is proving a transparent Air Quality information for more than 70 countries, covering more than 9000 stations in 600 major cities, via those two websites: [aqicn.org](http://aqicn.org) and [waqi.info](http://waqi.info).
+
+To use this binding, you first need to [register and get your API token](http://aqicn.org/data-platform/token/).
+
+## Supported Things
+
+There is exactly one supported thing type, which represents the air quality information for an observation location. It has the `aqi` id. Of course, you can add multiple Things, e.g. for measuring AQI for different locations.
+
+## Discovery
+
+There is no discovery implemented. You have to create your things manually.
+
+## Binding Configuration
+ 
+The binding has no configuration options, all configuration is done at Thing level.
+ 
+## Thing Configuration
+
+The thing has a few configuration parameters:
+
+| Parameter | Description                                                              |
+|-----------|------------------------------------------------------------------------- |
+| apikey    | Data-platform token to access the AQIcn.org service. Mandatory.            |
+| location  | Geo coordinates to be considered by the service. Mandatory. |
+| refresh   | Refresh interval in minutes. Optional, the default value is 60 minutes.  |
+
+For the location parameter, the following syntax is allowed:
+
+`37.8;-122.4`
+
+
+## Channels
+
+The AirQuality information that is retrieved is available as these channels:
+
+
+| Channel ID | Item Type    | Description              |   |
+|------------|--------------|------------------------- |---|
+| aqiLevel | Number | Air Quality Index | |
+| aqiDescription | String | AQI Description |
+| locationName | String | Nearest measuring station location |
+| locationUrl | String | Direct URL to the station results |
+| stationId | Number |
+| pm25 | Number | Fine particles pollution level (PM2.5) |
+| pm10 | Number | Coarse dust particles pollution level (PM10) |
+| no2 | Number | Nitrogen Dioxide level (NO2) |
+| co | Number | Carbon monoxide level (CO) |
+| observationTime | DateTime | Observation date and time |
+| temperature | Number | Temperature in Celsius degrees |
+| pressure | Number | Pressure level |
+| humidity | Number | Humidity level |
+
+`AQI Description` item provides a human-readable output that can be interpreted e.g. by MAP transformation:
+ 
+airquality.map
+```
+NO_DATA=No data
+GOOD=Good
+MODERATE=Moderate
+UNHEALTHY_FOR_SENSITIVE=Unhealthy for sensitive groups
+UNHEALTHY=Unhealthy
+VERY_UNHEALTHY=Very unhealthy
+HAZARDOUS=Hazardous
+```
+
+## Full Example
+
+airquality.things:
+
+```
+airquality:aqi:Krakow "AirQuality in Krakow" [ apikey="XXXXXXXXXXXX", location="50.06465,19.94498", refresh=60 ]
```

Comment:
I just discovered that you can add a Location in the thing description 😄 

Could you change the description to "AirQualitry" @ "Krakow" so we have examples demostrating this syntax?